frontend/src/features/rollout-announcement/components/NewFeatureContent.tsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useRef } from 'react'
+import { Box, BoxProps, ModalBody, ModalHeader, Text } from '@chakra-ui/react'
+import lottie from 'lottie-web'
+
+import { NewFeatureTag } from './NewFeatureTag'
+
+const TopSpacer = (): JSX.Element => (
+  <Box
+    width="100%"
+    height="4.5rem"
+    backgroundColor="primary.100"
+    borderTopRadius="1rem"
+  />
```

Comment:
looking at it, i still don't think this is needed. the appropriate padding and bg can be put in the `LottieAnimation` component

Reply:
Right, removed TopSpacer and used padding and bg instead